frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -583,10 +603,16 @@ pub mod pallet {
 		/// Duration of the unsigned phase.
 		#[pallet::constant]
 		type UnsignedPhase: Get<Self::BlockNumber>;
+
 		/// Duration of the signed phase.
 		#[pallet::constant]
 		type SignedPhase: Get<Self::BlockNumber>;
 
+		/// Minimum number of signed and unsigned blocks that EPM expects for a successful
```

Comment:
> Also what do you think of a name like `FallbackThresholdBlocks`

I thought of it as well. TBH, I think it's easier to reason about emergency phase rather than fallback throttling (especially because we don't even have fallback configured in Polkadot/Kusama). My reasoning for this (and how I tink about the EPM phase transitions) is that emergency phase is the real deal here. If an election fails, we start the emergency phase process by first trying the fallback. 

I will improve the documentation in any case, we can do better and ensure that it's documented that both fallback and emergency phase are throttled.